CHANGELOG.md

```diff
@@ -2,27 +2,94 @@
 
 All notable changes to Zebra are documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org).
 
-## [Zebra 1.0.0-rc.5](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.5) - 2023-02-TODO INSERT DATE HERE
+## [Zebra 1.0.0-rc.5](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.5) - 2023-02-23
 
-In this release we ... (TODO)
-We also check that Zebra is following the consensus chain each time it starts up.
+This release
+
+- finishes the implementation of mining-related RPCs;
+- makes Zebra ready for testing and initial adoption by mining pools;
+- adds support for the latest version of `lightwalletd`;
+- makes non-finalized chain forks instant;
+- contains other improvements such as deduplication of RedPallas code.
+
+Zebra now also checks that it is following the consensus chain each time it
+starts up.
 
 ### Security
-- Check that Zebra's state contains the consensus chain each time it starts up. This implements
-  the "settled network upgrade" consensus rule using all of Zebra's checkpoints ([#6163](https://github.com/ZcashFoundation/zebra/pull/6163)).
-  *User action required:*
-  - If your config is based on an old version of Zebra, or you have manually edited it,
-    make sure `consensus.checkpoint_sync = true`.
-    This option has been true by default since March 2022.
+
+- Check that Zebra's state contains the consensus chain each time it starts up.
+  This implements the "settled network upgrade" consensus rule using all of
+  Zebra's checkpoints
+  ([#6163](https://github.com/ZcashFoundation/zebra/pull/6163)).
+
+  _User action required:_
+
+  - If your config is based on an old version of Zebra, or you have manually
+    edited it, make sure `consensus.checkpoint_sync = true`. This option has
+    been true by default since March 2022.
+
+- Bump hyper from 0.14.23 to 0.14.24, fixing a denial of service risk ([#6094](https://github.com/ZcashFoundation/zebra/pull/6094))
+- Bump once_cell from 1.17.0 to 1.17.1 ([#6161](https://github.com/ZcashFoundation/zebra/pull/6161))
```

Comment:
I don't think there's a security fix in the once_cell version bump, looks like they just replaced `usize` memory addresses with `std::ptr`.

Reply:
I removed this entry in [8602571](https://github.com/ZcashFoundation/zebra/pull/6213/commits/8602571855d10543da17adc1607aebb83f400bf2), and I put it in the comment with the excluded PRs.